core/src/main/java/org/apache/accumulo/core/trace/TraceUtil.java

```diff
@@ -96,7 +97,10 @@ public static Span startServerRpcSpan(Class<?> caller, String spanName, TInfo ti
 
   private static Span startSpan(Class<?> caller, String spanName, SpanKind kind,
       Map<String,String> attributes, TInfo tinfo) {
-    if (!enabled && !Span.current().getSpanContext().isValid()) {
+    Context context = tinfo == null ? null : getContext(tinfo);
```

Comment:
Are you seeing `getContext` return null? Looking at the OpenTelemetry code, it doesn't look like it returns null, but I could be looking at the wrong version.

Reply:
I talked with @ctubbsii about this, and I think I understand the changes more. Based on our discussion, I think the changes here enable server tracing to occur when disabled on the server side, but requested by the client.